Create Demae Domino's Guide (#297)
RiiConnect24/Wii-Guide: _pages/en_US/wiilink-demae-dominos.md

---
title: WiiLink Demae Domino's Guide
---

{% include toc title="Table of Contents" %}

If you need help for anything regarding this tutorial, please join [the WiiLink Discord server](https://discord.gg/wiilink).
{: .notice--info}

In order to prevent abuse, a $1 USD paywall has been put in place. Due to this, a credit card is required. All purchases are handled by Stripe Inc. and will be refunded. Funds may take 5-10 business days to settle. For more information, read the [Terms of Service](https://demae.wiilink24.com/tos) and [Privacy Policy](https://demae.wiilink24.com/privacypolicy). 
{: .notice--info}

As this service places an order to Domino's Pizza, payment is required at the door. This is usually in the form of cash, although credit/debit card machines are possible if the delivery person has one.
{: .notice--info}

> Currently, only the United States of America and Canada are supported.

Reviewer: Currently we write "USA" everywhere so I think it is better to stay consistent.

{: .notice--warning}

> You cannot use the default Dolphin NAND with this service. You must use your own NAND dump.

Reviewer: I think it would be a good idea to describe briefly how to import one
something along the lines of "This can be done under tools -> manage nand -> import bootmii nand backup

{: .notice--warning}

#### What you need

* An SD card or USB drive

> * A Wii with an Internet connection

Reviewer: ```suggestion
* A Wii with an internet connection
```

* A computer
* A Discord account
 
#### Instructions

##### Section I - Obtaining your Wii's Console ID

1. Download the [Get Console ID](https://oscwii.org/library/app/GetConsoleID) homebrew app from the [Open Shop Channel](https://oscwii.org).

> 2. Unzip the downloaded file and copy the `apps` folder to your SD Card or USB.

Reviewer: ```suggestion
2. Unzip the downloaded file and copy the `apps` folder to your SD Card or USB drive.
```
Just keeping wording consistent

![Authorize Demae Domino's to access your Discord account](/images/Demae-Dominos/discord-oauth.png)
3. Click on the `Purchase Access` button in the top right corner. Please read the contents on screen very carefully before clicking the `Purchase` button.
4. You will be brought to a purchase page managed by Stripe Inc. Here you must enter your payment information along with the Wii Console ID from the last section.
![Purchase Page](/images/Demae-Dominos/purchase-page.png)
5. If the purchase succeeded, you will be brought to a success page. The WAD personalized for your Wii Console will be emailed to you. Please allow for up to 10 minutes for the email to arrive.
6. Once the email arrives, download the attached WAD file and move it to the `WAD` folder on your SD Card or USB.

##### Section III - Obtaining the Set Personal Data Channel

If you are using a Japanese Wii, this step can be skipped.
{: .notice--info}

If you already have the WiiLink Set Personal Data channel, you must install this one as it is updated to support addresses.
{: .notice--warning}

1. Click the download link for your console.
   * [Wii]()
   * [vWii]()

> 2. Move the downloaded WAD file to the `WAD` folder on your SD Card or USB.

Reviewer: ```suggestion
2. Copy the downloaded WAD file to the `WAD` folder on your SD Card or USB.
```

Is there a good reason to recommend "move" specifically? I suppose this is a one-time thing

Author: I guess it should be copy as this channel can be launched with Dolphin, file isn't too big either

##### Section IV - Installing WADs

You will now install the Food Channel and Set Personal Data channels.

1. Put your SD card or USB drive in your Wii.
2. Launch the Homebrew Channel on your Wii.
3. Launch Wii Mod Lite.

> 4. Using the +Control Pad on your Wii Remote, navigate to `WAD Manager`, and then navigate to the `wad` folder.

Reviewer: ```suggestion
4. Using the +Control Pad on your Wiimote, navigate to `WAD Manager`, and then navigate to the `wad` folder.
```

Perhaps it is worth consolidating this with the Wii Mod Lite page? Though, I understand this is somewhat time-sensitive and that page probably needs some review too. I don't want you to feel inclined to clean that up.

Author: I put this off for too long, maybe in the future we could consolidate.

Reviewer: don't worry about it, it's not your problem
